### PR TITLE
M Mykytovych SQL

### PR DESCRIPTION
**Pull request by marjana07**

### CODE REVIEW

File: src/test/java/orangetest/pages/VacanciesPage.java [hunk 1]
```diff
@@ -87,6 +94,17 @@ public boolean checkAddedVacancy() {
                         o.contains(vacancy.getVacancyName()));
     }
 
+    public boolean isHiringManager(String vacancyName, String hiringManager) {
```
Comment (nmelnyk-corevalue) [inline src/test/java/orangetest/pages/VacanciesPage.java]: isHiringManagerVisible?
isHiringManagerPresent?

File: src/test/java/orangetest/sqltask/ConnectionToDatabase.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package orangetest.sqltask;
+
+import java.sql.*;
+
+public class ConnectionToDatabase extends DataProvider {
```
Comment (nmelnyk-corevalue) [inline src/test/java/orangetest/sqltask/ConnectionToDatabase.java]: extends DataProvider makes no sense here

File: src/test/java/orangetest/sqltask/SqlDataProvider.java [hunk 3]
```diff
@@ -0,0 +1,136 @@
+package orangetest.sqltask;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class SqlDataProvider extends DataProvider {
```
Comment (nmelnyk-corevalue) [inline src/test/java/orangetest/sqltask/SqlDataProvider.java]: one SqlDataProvider for Emps and Vacancies breaks SRP - pliz split it into 2 providers

File: src/test/java/orangetest/sqltask/SqlDataProvider.java [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+                employee.setEmpLastName(rs.getObject("emp_lastname").toString());
+                return employee;
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return null;
+    }
+
+    public List<Employee> employeeSqlDataProvider() {
```
Comment (nmelnyk-corevalue) [inline src/test/java/orangetest/sqltask/SqlDataProvider.java]: public List<Employee> employeeSqlDataProvider() -> getEmployees() ?

File: src/test/java/orangetest/pages/VacanciesPage.java (hunk 1 above)
Comment (marjana07) [reply src/test/java/orangetest/pages/VacanciesPage.java]: fixed

File: src/test/java/orangetest/sqltask/ConnectionToDatabase.java (hunk 2 above)
Comment (marjana07) [reply src/test/java/orangetest/sqltask/ConnectionToDatabase.java]: fixed, moved fields to ConnectionToDatabase class

File: src/test/java/orangetest/sqltask/SqlDataProvider.java (hunk 3 above)
Comment (marjana07) [reply src/test/java/orangetest/sqltask/SqlDataProvider.java]: fixed

File: src/test/java/orangetest/sqltask/SqlDataProvider.java (hunk 4 above)
Comment (marjana07) [reply src/test/java/orangetest/sqltask/SqlDataProvider.java]: fixed

**Pull request merged**